policy-test/src/lib.rs

```diff
@@ -133,7 +133,22 @@ where
     let test = test(client.clone(), ns.name());
     let res = tokio::spawn(test.instrument(tracing::info_span!("test", ns = %ns.name()))).await;
     if res.is_err() {
-        // If the test failed, stop tracing so the log is not polluted with more
+        // If the test failed, list the state of all pods/containers in the namespace.
+        let pods = kube::Api::<k8s::Pod>::namespaced(client.clone(), &ns.name())
+            .list(&Default::default())
+            .await
+            .expect("Failed to get pod status");
+        for p in pods.items.into_iter() {
```

Comment:
nit: i'm surprised we can't just write
```suggestion
        for p in pods.items {
```
here?